frontend/src/components/pages/ComponentLibrary.tsx

```diff
@@ -1,15 +1,53 @@
 import React from "react";
+import { FormProvider, useForm } from "react-hook-form";
+import { HStack } from "@chakra-ui/react";
 
+import { Grade } from "../../APIClients/types/UserClientTypes";
+import { ClassroomForm } from "../../types/ClassroomTypes";
 import StudentDashboardHeader from "../assessments/assessment-creation/StudentDashboardHeader";
+import ClassroomCard from "../classrooms/ClassroomCard";
+import AddClassroomModal from "../user-management/student/AddClassroomModal";
+import AddStudentModal from "../user-management/student/AddStudentModal";
+
+import MobileRedirect from "./MobileRedirect";
+
+const defaultValues = {
+  className: "",
+  schoolYear: "",
+  gradeLevel: Grade.K,
+} as ClassroomForm;
 
 const ComponentLibrary = (): React.ReactElement => {
+  const methods = useForm<ClassroomForm>({
+    defaultValues,
+    mode: "onChange",
+  });
   return (
-    <div>
+    <FormProvider {...methods}>
       <StudentDashboardHeader
         assessmentName="Unit 0 Review Test"
         classroomName="Mathematics 4 - Mr. Roberts"
       />
-    </div>
+      <MobileRedirect />
+      <HStack justifyContent="center">
+        <ClassroomCard
+          activeAssessments={2}
+          assessmentCount={1}
+          grade={Grade.GRADE_4}
+          name="Counting and Numbers"
+          studentCount={23}
+        />
+        <ClassroomCard
+          activeAssessments={0}
+          assessmentCount={3}
+          grade={Grade.GRADE_8}
+          name="Sorting and Classifying"
+          studentCount={14}
+        />
+        <AddClassroomModal />
```

Comment:
maybe we dump this in `ClassroomsPage`? but doesn't really matter ig

Reply:
i think i will leave that for David to do :D